fin_control/README.md

```diff
@@ -1,9 +1,9 @@
-# Fin control package.
-The fin control:
+## `Fin control` node
+The `fin control` node:
```

Comment:
@patapufate lacking underscores in node names here and there:
```suggestion
The `fin_control` node:
```

Reply:
Fixed!